src/Form.tsx

```diff
@@ -72,7 +71,7 @@ const FormImpl: Form = (React.forwardRef(
 ) as unknown) as Form;
 
 FormImpl.displayName = 'Form';
-FormImpl.propTypes = propTypes;
+FormImpl.propTypes = propTypes as any;
```

Comment:
i wonder if we should do this more broadly. inferred props from `propTypes` don't play super well with explicit TS types

Reply:
Yeah this is so finicky :/